news/_posts/releases/2020-04-17-prestashop-1-7-7-0-beta-release.md

```diff
@@ -0,0 +1,150 @@
+---
+layout: post
+title:  "PrestaShop 1.7.7.0 Beta Is Ready To Be Tested!"
+subtitle: "You can download it and test it"
+date:   2020-04-17 12:00:00
+authors: [ PrestaShop ]
+image: /assets/images/2020/04/1.7.7_beta_banner.jpg
+icon: icon-leaf
+tags:
+ - 1.7.7
+ - version
+ - beta
+ - minor
+ - releases
+---
+
+The first Beta version for PrestaShop 1.7.7.0 is now ready for you to test!
+
+![Prestashop 1.7.7.0 BETA Release](/assets/images/2020/05/1.7.7_beta_banner.jpg)
+
+After more than one year in the making, we are really thrilled to announce that we are releasing the 1.7.7.0 beta. This version is even bigger than the previous one with 957 merged pull requests, compared to 600 for 1.7.6.0 (our previous record) – and we are still at least a month away from the final version. 
```

Comment:
There were 748 merged PRs in 1.7.6.0

```suggestion
After more than one year in the making, we are really thrilled to announce that we are releasing the 1.7.7.0 beta. This version is even bigger than the previous one with 957 merged pull requests, compared to 748 for 1.7.6.0 (our previous record) – and we are still at least a month away from the final version. 
```

Reply:
748 is the total until today ? 600 is the number until the build of the beta

Reply:
The number is regarding the final version as announced [here](https://build.prestashop.com/news/prestashop-1-7-6-0-available/#changes). I think we can safely use it as it _was_ our record and we beat it by far 😄

Reply:
Ahh yes, for 1.7.6, my bad